## Targeted Sequencing

These are properties for Targeted Sequencing that are non-core, and thus not found in all sequencing experiments. 

Some will be unique to this type of experiment and some common to several types.

Table of Targeted Sequencing properties

| Field name | Specificity | Definition | Mandatory | Example | Type | Controlled Vocab Terms | Comment | INSDC | IHEC (International Human<br>Epigenome Consortium) |  | FAIRE (eDNA) | DwC (eDNA) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| library_selection | specific | Method used to enrich the target in the sequence library preparation. [INSDC] | TRUE | unspecified | string | RANDOM, PCR, RANDOM PCR, RT-PCR, HMPR, MF, repeat fractionation, size fractionation, MSLL, cDNA, cDNA_randomPriming, cDNA_oligo_dT, PolyA, Oligo-dT, Inverse rRNA, Inverse rRNA selection, ChIP, ChIP-Seq, MNase, DNase, Hybrid Selection, Reduced Representation, Restriction Digest, 5-methylcytidine antibody, MBD2 protein methyl-CpG binding domain, CAGE, RACE, MDA, padlock probes capture method, other, unspecified | This property belongs to "Targeted Sequencing" category. Example 1) of random in. https://www.ebi.ac.uk/ena/browser/view/SRX24644330 <br>2)TODAY ProDIGY Exome Sequencing Study https://www.ebi.ac.uk/ena/browser/view/PRJNA432773 but is in dGaP<br>3) Knockout mice with pituitary malformations help identify human cases of hypopituitarism https://www.ebi.ac.uk/ena/browser/view/PRJNA1032055<br>4) RELEVANT  Whole exome sequencing of canine mammary tumors  https://www.ebi.ac.uk/ena/browser/view/PRJNA489159 and https://www.ebi.ac.uk/ena/browser/view/SRX4636157<br>5) https://pmc.ncbi.nlm.nih.gov/articles/PMC11419970/<br>Library preparation for whole exome sequencing was performed using the Sureselect Human All Exon V6 kit (Illumina Inc) and sequencing was performed with NovaSeq6000 150PE 18 Gb/sample (Marcogen Inc) (18). Sequencing data were processed using HD Genome One bioinformatics software (DREAMgenics, Oviedo, Spain), certified with IVD/CE marking as described previously (19). The process included quality control and alignment, somatic variant calling, variant annotation, and copy number neutral loss of heterozygosity (CN‐LOH) identification. <br>6) https://pmc.ncbi.nlm.nih.gov/articles/PMC11372244/<br>Human genomic DNA was sheared into approximately 150–200 base-pair fragments using a S220 Focused-Ultrasonicator (Covaris, Woburn, Massachusetts) according to the manufacturer’s instructions. Exome capture and library preparation were carried out using SureSelectXT Human All Exon V6 + UTR (Agilent Technologies, Santa Clara, CA). The prepared library was sequenced on a NovaSeq 6,000 System (Illumina, San Diego, CA) with 150 base-pair reads, producing up to 12 Gb of data per sample. <br>7) https://pubmed.ncbi.nlm.nih.gov/39495750/ <br>"Multiple commercially available kits are used for WES, which employ similar protocols for target enrichment. These protocols typically involve hybridization between exon sequences and biotin-conjugated DNA or RNA probes, followed by capture via streptavidin-coated magnetic beads"<br>see Fig 1. The design and workflow of the current study. | library_selection |  |  |  |  |
| experiment target regions |  |  |  |  |  |  |  |  |  |  |  |  |
| target_gene |  | Targeted gene or locus name for marker gene studies | FALSE | COI | string |  |  |  |  |  | targetTaxonomicAssay | targetTaxonomicAssay |
| target_subfragment |  | Name of subfragment of a gene or locus. Important to e.g. identify special regions on marker genes like V6 on 16S rRNA. | FALSE | V6 | string |  |  | target_gene |  |  | targetTaxonomicScope | targetTaxonomicScope |
| targetTaxonomicAssay |  | Taxa or species name targeted by the primers, probes, and/or other approaches applied in the PCR. | TRUE | 83 | string |  | only mandatory for metabarcoding | target_subfragment |  |  | target_gene |  |
| targetTaxonomicScope |  | The taxonomic group(s) targeted in the study. This can differ from the targetTaxonomicAssay. For example, the targetTaxonomicAssay may be Chordata while the targetTaxonomicScope is Chondrichthyes (bony fish, sharks and rays). | FALSE | teleost | string |  |  |  |  |  | target_subfragment |  |
| barcoding_pcr_appr |  | PCR approach for metabarcoding | FALSE | two-step PCR | string |  | only specific for metabarcoding |  |  |  | barcoding_pcr_appr |  |
| pcr_primer_forward |  | Forward PCR primer (5' - 3') that were used to amplify the sequence of the targeted gene. The primer sequence should NOT contain MIDs and adapter sequences, and should be reported in uppercase letters. If multiple forward primers are present in a single PCR reaction, either report them separately in separate submissions (with separate OTU tables for each primer set), or list the primers here separated by pipe ( \| ) and provide one OTU table resulting from all the primers applied here. The latter is recommended only when the multiple primers target the same gene region and thus a single OTU table was generated from the multiplexed PCR. This term is required except for PCR free methods. | TRUE | GGWACWGGWTGAACWGTWTAYCCYCC | string | IUPAC | could be used in metabarcoding or other types of sequencing |  |  |  | pcr_primer_forward |  |
| pcr_primer_reverse |  | Reverse PCR primer (5' - 3') that were used to amplify the sequence of the targeted gene. The primer sequence should NOT contain MIDs and adapter sequences, and should be reported in uppercase letters. If multiple reverse primers are present in a single PCR reaction, either report them separately in separate submissions (with separate OTU tables for each primer set), or list the primers here separated by pipe ( \| ) and provide one OTU table resulting from all the primers applied here. The latter is recommended only when the multiple primers target the same gene region and thus a single OTU table was generated from the multiplexed PCR. This term is required except for PCR free methods. | TRUE | GGRGGRTASACSGTTCASCCSGTSCC | string | IUPAC | could be used in metabarcoding or other types of sequencing |  |  |  | pcr_primer_reverse |  |
| pcr_primer_name_forward |  | Name of the pcr_primer_forward. If multiple primers are listed in pcr_primer_forward, list the names respectively, separated by pipe ( \| ). | FALSE | mlCOIintF | string |  | could be used in metabarcoding or other types of sequencing |  |  |  | pcr_primer_name_forward |  |
| pcr_primer_name_reverse |  | Name of the pcr_primer_reverse. If multiple primers are listed in pcr_primer_reverse, list the names respectively, separated by pipe ( \| ). | FALSE | mlCOIintR | string |  | could be used in metabarcoding or other types of sequencing |  |  |  | pcr_primer_name_reverse |  |
| pcr_primer_reference_forward |  | Reference (i.e. DOI) of the pcr_primer_forward. If multiple primers are listed in pcr_primer_forward, list the references respectively, separated by pipe ( \| ). | FALSE | https://doi.org/10.1186/1742-9994-10-34 | string |  | could be used in metabarcoding or other types of sequencing |  |  |  | pcr_primer_reference_forward |  |
| pcr_primer_reference_reverse |  | Reference (i.e. DOI) of the pcr_primer_reverse. If multiple primers are listed in pcr_primer_reverse, list the references respectively, separated by pipe ( \| ). | FALSE | https://doi.org/10.1186/1742-9994-10-34 | string |  | could be used in metabarcoding or other types of sequencing |  |  |  | pcr_primer_reference_reverse |  |
| nucl_acid_ext_kit |  | The name of the extraction kit used to recover the nucleic acid fraction of an input material is performed | FALSE | QIAamp DNA Blood Mini Kit | string |  | this most be more widely used? how useful though? |  |  |  |  |  |
| library_selection_details |  | Information on the library selection details including kit. | FALSE | individual hybridisation; SureSelect XT Libray | string |  | only specific for exome_sequencing or similar |  |  |  |  |  |
| library_capture_details |  | High-level information on the probes etc.  including the kit used to capture the exome DNA | TRUE | Canine All Exon V2; 16-24 hours hybridisation | string |  | only specific for exome_sequencing or similar |  |  |  |  |  |
